### PR TITLE
V1.1.0 release

### PR DESCRIPTION
**Pull request by haixiw**
*Issue #, if available:*

*Description of changes:*

 release TGI 1.1.0 , 
      add new Dockerfile and new license generated from scallion

Test:
this change has been tested by the github pipeline here https://github.com/awslabs/llm-hosting-container/blob/main/.github/workflows/build-huggingface.yml#L90

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu [hunk 1]
```diff
@@ -223,8 +223,6 @@ RUN HOME_DIR=/root && \
 
 # From Scallion open-source dependencies
 COPY llm-hosting-container/huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES /root/THIRD-PARTY-LICENSES
-# From generate_oss_compliance.sh
-COPY llm-hosting-container/huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/PYTHON_PACKAGES_LICENSES /root/PYTHON_PACKAGES_LICENSES
```
Comment (jinyoung-lim) [inline huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:227]: Why is this needed? I.e. Why is this change to 1.0.3 needed?

**Comment by jinyoung-lim** [comment]
General question: How did we test this? Is this something that HF provides and we just make a PR? Or was there some dev work involved for testing and verification?

**Review by jinyoung-lim — COMMENTED** [review]
General question: How did we test this? Is this something that HF provides and we just make a PR? Or was there some dev work involved for testing and verification?

**Comment by haixiw** [comment]
> General question: How did we test this? Is this something that HF provides and we just make a PR? Or was there some dev work involved for testing and verification?

I just found out that this line https://github.com/awslabs/llm-hosting-container/pull/34/files#diff-99e58afe5cc09ca032494c7b0d4aad4a4439e65c478df4d5a778dd5c1bcf350fL221 already generated this python dependency license file in the image. So we don't need to import manually here. Thanks

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu (hunk 1 above)
Comment (haixiw) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:227]: I just found out that this line https://github.com/awslabs/llm-hosting-container/pull/34/files#diff-99e58afe5cc09ca032494c7b0d4aad4a4439e65c478df4d5a778dd5c1bcf350fL221 already generated this python dependency license file in the image. So we don't need to import manually here. Thanks

**Comment by haixiw** [comment]
> General question: How did we test this? Is this something that HF provides and we just make a PR? Or was there some dev work involved for testing and verification?

We tested here https://github.com/awslabs/llm-hosting-container/actions/runs/6342812714/job/17231890926 so basically there's a github pipeline running the tests. I triggered it for the test branch previously

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu (hunk 1 above)
Comment (whittech1) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:227]: Even with that being the case, I am wary of modifying past image versions. Will it be sufficient to just omit this line in versions >=1.1.0 ?

File: .github/workflows/build-huggingface.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ on:
       tgi-version:
         description: 'tgi version'
         required: true
-        default: '1.0.3'
```
Comment (whittech1) [inline .github/workflows/build-huggingface.yml:9]: non-blocking: could you please add a runbook for this process so that others on the team can help with this type of work?

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu [hunk 3]
```diff
@@ -223,8 +223,6 @@ RUN HOME_DIR=/root && \
 
 # From Scallion open-source dependencies
 COPY llm-hosting-container/huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES /root/THIRD-PARTY-LICENSES
-# From generate_oss_compliance.sh
```
Comment (whittech1) [inline huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:226]: Why are we modifying an image version which has already been released? Wouldn't it be safer to keep 1.0.3 as-is?

File: .github/workflows/build-huggingface.yml (hunk 2 above)
Comment (haixiw) [reply .github/workflows/build-huggingface.yml:9]: Sure thing. Thanks

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu (hunk 3 above)
Comment (haixiw) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu]: ommited in new commit

**Review by whittech1 — APPROVED** [review]
